tchannel-example/src/main/java/com/uber/tchannel/thrift/SetValueHandlerDefault.java

```diff
@@ -38,17 +35,13 @@ public SetValueHandlerDefault(Map<String, String> keyValueStore) {
         this.keyValueStore = keyValueStore;
     }
 
-    @Override
-    public Response<KeyValue.setValue_result> handle(Request<KeyValue.setValue_args> request) {
+    public KeyValue.setValue_result handleImpl(KeyValue.setValue_args setValueArgs) {
```

Comment:
Not going to Fly, we need to pass in everything from the Request to the Handle implementation e.g. Trace, Transport Headers, etc.